pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: gallery
 description: A resource to help developers evaluate and use Flutter.
 repository: https://github.com/flutter/gallery
-version: 2.5.0+020500
+version: 2.5.0+020600
```

Comment:
You need to make the first part match the second part:

`2.6.0+020600`

Reply:
Oh I totally missed that, thanks.  Will do.